rest/importtest/import_test.go

```diff
@@ -2378,3 +2379,75 @@ func TestImportUpdateExpiry(t *testing.T) {
 		})
 	}
 }
+
+func TestPrevRevNoPopulationImportFeed(t *testing.T) {
+	base.SkipImportTestsIfNotEnabled(t)
+	if base.UnitTestUrlIsWalrus() {
+		t.Skipf("test requires CBS for previous rev no assertion")
+	}
+
+	rtConfig := rest.RestTesterConfig{
+		DatabaseConfig: &rest.DatabaseConfig{DbConfig: rest.DbConfig{
+			AutoImport: true,
+		}},
+	}
+
+	rt := rest.NewRestTester(t, &rtConfig)
+	defer rt.Close()
+	dataStore := rt.GetSingleDataStore()
+	ctx := base.TestCtx(t)
+
+	if !rt.Bucket().IsSupported(sgbucket.BucketStoreFeatureMultiXattrSubdocOperations) {
+		t.Skip("Test requires multi-xattr subdoc operations, CBS 7.6 or higher")
+	}
+
+	// Create doc via the SDK
+	mobileKey := t.Name()
+	mobileBody := make(map[string]interface{})
+	mobileBody["channels"] = "ABC"
+	_, err := dataStore.Add(mobileKey, 0, mobileBody)
+	assert.NoError(t, err, "Error writing SDK doc")
+
+	// Wait for import
+	err = rt.WaitForCondition(func() bool {
+		return rt.GetDatabase().DbStats.SharedBucketImportStats.ImportCount.Value() == 1
+	})
+	require.NoError(t, err)
```

Comment:
```suggestion
	base.RequireWaitForStat(t, func() int64 {
		return rt.GetDatabase().DbStats.SharedBucketImportStats.ImportCount.Value()
	}, 0)
```

This will print the value if it doesn't match.